packages/platform-sdk/test/integration-tests/sdk-v3/middlewares.test.ts

```diff
@@ -25,6 +26,9 @@ import {
 import { createApiBuilderFromCtpClient } from '../../../src'
 import { randomUUID } from 'crypto'
 
+import * as matchers from 'jest-extended'
+expect.extend(matchers)
```

Comment:
Why it needs to be imported again? I see you already imported this in `setup.js` above.

Reply:
We need it for the typescript global compiler, and IDE support, if I remove or comment off the setup.js file, vs code will complain.